include/evm_runtime/tables.hpp

```diff
@@ -149,6 +157,11 @@ struct [[eosio::table("inevm")]] [[eosio::contract("evm_contract")]] balance_wit
     }
 
     balance_with_dust& operator-=(const intx::uint256& amount) {
+
+        check(balance.symbol != eosio::symbol(), "symbol can't be empty in balance_with_dust");
+        intx::uint256 minimum_natively_representable = pow10_const(evm_precision - balance.symbol.precision());
+        uint64_t min_asset = minimum_natively_representable[0];
+
```

Comment:
Is there any value in caching the `minimum_natively_representable` value in the config table also to not have to calculate all the time?

Reply:
please review again. thanks!